.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    # should we check? what about merge commits?
+    # if: "contains(github.event.head_commit.message, 'version')"
```

Comment:
could also do the same check that is in the `release tag` here instead, similar to this https://github.com/marketplace/actions/skip-based-on-commit-message#just-use-bash

Reply:
Yea this isn’t a bad idea